### PR TITLE
Fix store_benchmark_outputs file path 

### PR DESCRIPTION
**Pull request by berroar**
## Context

The file path for the store_benchmark_outputs python script needs to be updated in docker_entrypoint.sh as this file has been moved to the scripts directory.

## How to review
Check that the script is being called correctly

### CODE REVIEW

**Comment by berroar** [comment]
[Fix daily benchmark script](https://trello.com/c/g1Nqdu7U/3636-fix-daily-benchmark-script)

**Comment by berroar** [comment]
> If the locust filenames have changed then you need to update all the old references. Ie in `visualise_results.py` etc you need to make it work for both filenames so that the old data can be used as well as the new ones.

I have now updated the visualise_results.py so that it works for the old and new file names.

**Comment by JamesGardiner** [comment]
`docs/performance_investigations still has a reference to the `baseline_distribution.csv`

File: ci/run-benchmark.yaml [hunk 1]
```diff
@@ -18,6 +18,7 @@ params:
   PARALLELISM:
   TIMEOUT:
   OUTPUT_DIR:
+  SERVICE_ACCOUNT_JSON: ((gcp.service_account.json))
```
Comment (MebinAbraham) [inline ci/run-benchmark.yaml]:
```suggestion
  SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
```

File: doc/performance-investigations/README.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ More in-depth configuration is explained in the top-level [Readme](/README.md).
 You can use the `extract_distribution_summary.py` script to extract high level metrics from the Locust distributions CSV:
 
 ```
-python scripts/extract_distribution_summary.py < baseline_distribution.csv
+python scripts/extract_distribution_summary.py < baseline_stats.csv
```
Comment (MebinAbraham) [inline doc/performance-investigations/README.md]: Worth renaming this file.

File: runner_benchmark/taskset.py [hunk 3]
```diff
@@ -14,6 +14,9 @@
 
 
 class SurveyRunnerTaskSet(TaskSet, QuestionnaireMixins):
+    wait_time = between(int(os.getenv('USER_WAIT_TIME_MIN_SECONDS', 1)),
+                        int(os.getenv('USER_WAIT_TIME_MAX_SECONDS', 2)))
```
Comment (MebinAbraham) [inline runner_benchmark/taskset.py]: So this waits between the task executions not requests therefore we need to reintroduce the manual wait time in `replay_requests` method.

**Pull request merged**